src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -98,6 +98,15 @@ protected ActionResult execute() {
                                                         new Text(emailContent),
                                                         null);
             data.emailToEdit.emailId = emailId;
+        } else {
+            if (addressModeOn) {
+                statusToAdmin = "Email sent to " + addressReceiverListString;
```

Comment:
Technically, this is not the place where the email is sent as it just sends the job to the email sending queue.
The log message is not correct.
I would suggest to put the log message things inside `recordNewSentEmail` rather than having multiple if else to see whether it is `addressMode` or `groupMode`.